bin/sem-apply

```diff
@@ -32,7 +32,7 @@ begin
   puts "Upgrading schema for #{db.url}"
   count = util.apply!("./scripts")
   if count == 0
-    puts "  All scripts have been previously applied"
+    puts "  No scripts to apply"
```

Comment:
I think the previous wording is a bit more accurate - ie the common case is there are scripts in the directory but all have been applied. No scripts to apply really means no unapplied scripts found.

Reply:
Sounds good. It is probably preferable for the regular case but it is surprising in case of an abort.
In any way, I reverted it.